.gitignore

```diff
@@ -17,3 +17,6 @@ bin/
 
 # IDE
 .idea
+
+# MacOS
+.DS_Store
```

Comment:
Could you please add a new line to all the files with the error `No newline at end of file`?